CONTRIBUTING.md

```diff
@@ -4,25 +4,43 @@ Library issues are tracked using [GitHub Issues](https://github.com/stkent/bugsh
 
 Always review open issues before opening a new one. If you would like to work on an existing issue, please comment to that effect and assign yourself to the issue.
 
+# Commons Tasks
+
+## Library Module
+
+<a name="checks"></a> Run all checks:
+
+```shell
+./gradlew :bugshaker:checkLocal
+```
+
+Publish to your local Maven repository:
+
+```shell
+./gradlew :bugshaker:install
+```
+
+## Example App Module
+
+Assemble and deploy to an attached device:
+
+```shell
+./gradlew :example:assembleDebug && adb install -r example/build/outputs/apk/example-debug.apk
```

Comment:
Or simply `./gradlew installDebug` - the `installDebug` task depends on `assembleDebug`, and `example` is the only project that can be deployed. Besides being shorter, it has the advantage that it will also work if `adb` is not available on `PATH` (although it should be there in most cases).